docs/3.0rc/develop/write-tasks/use-task-runners.mdx

```diff
@@ -51,127 +38,104 @@ Within the restrictions of resources available, parallel execution can run tasks
-instead of using a specified task runner.
-</Note>
+Prefect future objects must be resolved explicitly before returning from your flow. 
+This can be done by calling `.wait()` on the future.
 
-For example, you can use `ConcurrentTaskRunner` to allow tasks to switch when they would block.
+For example:
 
-```python hl_lines="2 11"
+```python
 from prefect import flow, task
-from prefect.task_runners import ConcurrentTaskRunner
+from prefect.task_runners import ThreadPoolTaskRunner
 import time
 
+
 @task
 def stop_at_floor(floor):
     print(f"elevator moving to floor {floor}")
     time.sleep(floor)
     print(f"elevator stops on floor {floor}")
 
-@flow(task_runner=ConcurrentTaskRunner())
+
+@flow(task_runner=ThreadPoolTaskRunner(max_workers=3))
 def elevator():
+    floors = []
+
     for floor in range(10, 0, -1):
-        stop_at_floor.submit(floor)
-```
+        floors.append(stop_at_floor.submit(floor))
 
-If you specify an uninitialized task runner class, a task runner instance of that type 
-is created with the default settings. You can also pass additional configuration parameters 
-for task runners that accept parameters, such as [`DaskTaskRunner`](https://prefecthq.github.io/prefect-dask/) 
-and [`RayTaskRunner`](https://prefecthq.github.io/prefect-ray/).
+    [floor.wait() for floor in floors]
+```
 
 <Tip>
 **Default task runner**
 If you don't specify a task runner for a flow and you call a task with `.submit()` within the flow, 
-Prefect uses the default `ConcurrentTaskRunner`.
+Prefect uses the default `ThreadPoolTaskRunner`.
 </Tip>
-## Running tasks sequentially
-
-Sometimes, it's useful to force tasks to run sequentially to make it easier to reason about the 
-behavior of your program. Calling your tasks directly without using `.submit()` will automatically execute your tasks
-in order.
-
-<Note>
-In Prefect 3.0 `SequentialTaskRunner` is no longer necessary.
-</Note>
-The following example demonstrates tasks which run sequentially. 
-In the example, the flow `glass_tower` runs the task `stop_at_floor` for floors 1 through 38, in that order.
-
-```python
-from prefect import flow, task
-import random
-
-@task
-def stop_at_floor(floor):
-    situation = random.choice(["on fire","clear"])
-    print(f"elevator stops on {floor} which is {situation}")
-
-@flow(name="towering-infernflow",
-      )
-def glass_tower():
-    for floor in range(1, 39):
-        stop_at_floor(floor)
-    
-glass_tower()
-```
 
 ## Use multiple task runners
 
-Each flow can only have a single task runner, but sometimes you may want a subset of your tasks to run 
-using a specific task runner. In this case, you can create [subflows](/3.0rc/develop/write-flows/#composing-flows) 
+Each flow can only have one task runner, but sometimes you may want a subset of your tasks to run 
+using a different task runner than the one configured on the flow. 
+In this case, you can create [subflows](/3.0rc/develop/write-flows/#composing-flows) 
```

Comment:
```suggestion
In this case, you can create [nested flows](/3.0rc/develop/write-flows/#composing-flows) 
```

Reply:
Ah yes, love this

Reply:
Cool. Got this [PR](https://github.com/PrefectHQ/prefect/pull/14122/) open that updates everywhere to match @billpalombi's suggested usage.